README.md

```diff
@@ -99,6 +99,14 @@ python -m pip install --upgrade pip
 
 Install pytorch with CUDA (this repo has been tested with CUDA 11.7 and CUDA 11.8) and [tiny-cuda-nn](https://github.com/NVlabs/tiny-cuda-nn)
 
+NOTE: On Windows, PyTorch 2.0 is not currently supported; please install PyTorch 1.13.1 with CUDA 11.7:
```

Comment:
@machenmusik are you sure?

Reply:
I meant with Nerfstudio, mainly because I didn't get it to work with 2.0.1 when I tried it, not that it can't work